lib/generators/active_fedora/noid/install_generator.rb

```diff
@@ -0,0 +1,36 @@
+module ActiveFedora
+  module Noid
+    class InstallGenerator < Rails::Generators::Base
+      include Rails::Generators::Migration
+      source_root File.expand_path('../templates', __FILE__)
+
+      desc <<-END_OF_DESC
+Copies DB migrations
+      END_OF_DESC
+
+      # Implement the required interface for Rails::Generators::Migration.
+      # taken from http://github.com/rails/rails/blob/master/activerecord/lib/generators/active_record.rb
+      def self.next_migration_number(path)
+        if @prev_migration_nr
+          @prev_migration_nr += 1
+        else
+          last_migration = Dir[File.join(path, '*.rb')].sort.last
+          @prev_migration_nr = if last_migration
+                                 last_migration.sub(File.join(path, '/'), '').to_i + 1
+                               else
+                                 Time.zone.now.utc.strftime("%Y%m%d%H%M%S").to_i
+                               end
+        end
+        @prev_migration_nr.to_s
+      end
+
+      def banner
+        say_status('info', 'Installing ActiveFedora::Noid', :blue)
+      end
+
+      def migrations
```

Comment:
We can't do `active_fedora-noid` anything, because it is not valid in Rails project namespace.  We use `engine_cart/rake_task` not `rails/tasks/engine.rake`, so engine gem itself doesn't get `app:active_fedora_noid_engine:install:migrations` (and the 39 other :app tasks).  But in the test app, I do find a job under `active_fedora_noid_engine:install:migrations`, so I can give that a try.